File: dawn/src/main/java/com/lzq/dawn/util/time/TimeConstants.java
package com.lzq.dawn.util.time;

import androidx.annotation.IntDef;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

/**
 * @Name :TimeConstants
 * @Time :2022/8/31 15:43
 * @Author :  Lzq
 * @Desc :
 */
public class TimeConstants {

    public static final int MSEC = 1;
    public static final int SEC  = 1000;
    public static final int MIN  = 60000;
    public static final int HOUR = 3600000;
    public static final int DAY  = 86400000;

    @IntDef({MSEC, SEC, MIN, HOUR, DAY})
    @Retention(RetentionPolicy.SOURCE)
    public @interface Unit {
    }
}

File: dawn/src/main/java/com/lzq/dawn/util/time/TimeConstants.kt
package com.lzq.dawn.util.time

import androidx.annotation.IntDef

/**
 * @Name :TimeConstants
 * @Time :2022/8/31 15:43
 * @Author :  Lzq
 * @Desc :
 */
object TimeConstants {
    const val MSEC = 1
    const val SEC = 1000
    const val MIN = 60000
    const val HOUR = 3600000
    const val DAY = 86400000

    @IntDef(MSEC, SEC, MIN, HOUR, DAY)
    @Retention(AnnotationRetention.SOURCE)
    annotation class Unit
}